deploy/osps/default/osp-amzn2.yaml

```diff
@@ -25,6 +25,23 @@ spec:
     - name: "aws"
 
   bootstrapConfig:
+    templates:
+      configureProxyScript: |-
+        {{- if .HTTPProxy }}
+        cat <<EOF | tee -a /etc/environment
+        HTTP_PROXY={{ .HTTPProxy }}
+        http_proxy={{ .HTTPProxy }}
+        HTTPS_PROXY={{ .HTTPProxy }}
+        https_proxy={{ .HTTPProxy }}
+        {{- end }}
+        {{- if .NoProxy }}
+        NO_PROXY={{ .NoProxy }}
```

Comment:
In this case, you also need to add `cat <<EOF | tee -a /etc/environment` along with `EOF` at the end.

Reply:
Same goes for other operating systems.

Reply:
Or you can move `if .HTTPProxy` after `cat ...`.

Reply:
Good catch, fixed. 👍